### ISSUE TITLE
Add publish changes to the commits view

### ISSUE THREAD

**Issue by karczewskiPiotr** [vnLab-Lodz/PubLab#236]
Since each editor will be working on a separate branch their default commits will be made to those branches. Therefore, there is a need to some sort of `Publish changes` button in the commits view. A tooltip explaining what it does would also probably be good to have as well a confirmation for the action. 

Before the editor branch is merged into `main` a check should be performed to see if the editor is behind the `main` branch. In case it is the `main` should be first merged into the editor branch. Only after that can the editor branch be merged into `main`. This is so that any conflicts are handled on editor branches instead of `main`.
